cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractOperator.java

```diff
@@ -186,33 +182,33 @@ public final Future<Void> reconcile(Reconciliation reconciliation) {
         Timer.Sample reconciliationTimerSample = Timer.start(metrics().metricsProvider().meterRegistry());
 
         Future<Void> handler = withLock(reconciliation, LOCK_TIMEOUT_MS, () ->
-            resourceOperator.getAsync(namespace, name)
-                .compose(cr -> cr != null ? reconcileResource(reconciliation, cr) : reconcileDeletion(reconciliation)));
+                resourceOperator.getAsync(namespace, name)
+                        .compose(cr -> cr != null ? reconcileResource(reconciliation, cr) : reconcileDeletion(reconciliation)));
 
         Promise<Void> result = Promise.promise();
         handler.onComplete(reconcileResult ->
-            callSafely(reconciliation, () -> handleResult(reconciliation, reconcileResult, reconciliationTimerSample))
-                .onComplete(handleSafely(reconciliation, ignored -> result.handle(reconcileResult))));
+                callSafely(reconciliation, () -> handleResult(reconciliation, reconcileResult, reconciliationTimerSample))
+                        .onComplete(handleSafely(reconciliation, ignored -> result.handle(reconcileResult))));
 
         return result.future();
     }
 
     /**
      * Reconcile assembly resources in the namespace given by {@code reconciliation} having the name
      * give by {@code reconciliation}.
-     *
+     * <p>
```

Comment:
I guess you fixed this based on some IDE warning? I personally ignore this one because I think the Javadocs are more useful in the IDE than as HTML, so I don't think it really makes sense. But I'm fine if you wanna fix those. But in general, it is much better to create a separate PR for that and not mix it into another PR. It makes reviewing easier but also tracing changes in the code. The same applies to all the formatting and whitespaces changes unrelated to the PR.